packages/host/src/components/Login/Login.tsx

```diff
@@ -29,6 +30,11 @@ const StoreAutocompleteInput: React.FC<
 
 export const Login: React.FC = ({}) => {
   const t = useTranslation('app');
+  const { setPageTitle } = useHostContext();
+  useEffect(() => {
+    setPageTitle(`${t('app.login')} | ${t('app')} `);
+  }, []);
```

Comment:
Special one for "Login" page since it's outside the authenticated "Site" component